src/Groups/GAPGroups.jl

```diff
@@ -748,9 +947,10 @@ function sylow_subgroup(G::GAPGroup, p::Int64)
 end
 
 """
-    hall_subgroup(G::Group, P::Array{Int64})
+    hall_subgroup(G::Group, P::Vector{Int64})
 
-Return a Hall `P`-subgroup of `G`. It works only if `G` is solvable.
+Return a Hall `P`-subgroup of `G`.
```

Comment:
A definition of "Hall subgroup" would be nice, possibly taken from the GAP manual. Likewise for an example (again we could borrow from the GAP manual).

Reply:
Yes.